src/resources/grammar/SearchDSL.jison

```diff
@@ -2,7 +2,9 @@
 /*              common filter description language.                           */
 
 %{
-const {Merge, Operator} = require('../../js/utils/DSLParserUtil');
+const DSLParserUtil = require('../../js/utils/DSLParserUtil');
```

Comment:
Sorry I meant to reply to this — we're actually good to use `const` http://caniuse.com/#search=const